### ISSUE TITLE
Editorial workflow always needs to go through the PR process? No direct publishing possible?

### ISSUE THREAD

**Issue by laurenskling** [netlify/netlify-cms#3401]
I've just started using the editorial_workflow (GitLab), which looks awesome by the way.

Now i'm a bit confused at to how it should work with already published content. I can only press the save button, which creates a branch and pull(/merge) request. Now I can publish (merge the PR). So for my small change, 2 commits are made: the change commit and the merge commit. 2 pipelines running. It feels a bit overhead to always having to go this route.

So long story short: how can I make a direct commit on master while in editorial workflow?

**Comment by erezrokah** [comment]
Hi @laurenskling, you could change the merge method to squash:
https://www.netlifycms.org/docs/beta-features/#squash-merge-github-pull-requests

Not properly documented for GitLab and BitBucket, but should work

**Comment by laurenskling** [comment]
Thanks for the information @erezrokah , we'll just have to get used to it.
